spec.md

```diff
@@ -0,0 +1,121 @@
+# Specification
+
+Implement in your favorite languages by following this specification.
+
+* Date: 2015-09-07
+* Authors: Hsiaoming Yang, Jiangge Zhang
+
+
+## Interface
+
+Namespace should always be **GB2260**.
+
+## Data Structure
+
+There is a standard structure named `Division`. Those following properties
+should be included in it.
+
+- `code`: (str) The six-digit number of the specific administrative division.
+- `name`: (str) The Chinese name of the specific administrative division.
+- `year`: (str) Optional. The revision year, and empty means "latest".
+
+For example, a county of Beijing in JavaScript is defined like this:
+
+```javascript
+{
+  "code": "110103",
+  "name": "崇文区",
+  "year": "2003"
+}
+```
+
+The administrative level representation depends on recursion records. The
+division codes `XX0000` (`XX` is non-zero digits) mean provinces. And there are
```

Comment:
According to the context, `XX` is a plural form, the correct wording may be:

> `XX` **are** non-zero digit**s**

Actually, this makes a lot more sense to me:

> `X` is non-zero digit